options/ansi/generic/ctype-stubs.cpp

```diff
@@ -254,9 +254,48 @@ wctype_t wctype(const char *cs) {
 	return ct_null;
 }
 
-int iswctype(wint_t, wctype_t) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+// Taken from musl. See musl for the license/copyright!
+#define WCTYPE_ALNUM  1
+#define WCTYPE_ALPHA  2
+#define WCTYPE_BLANK  3
+#define WCTYPE_CNTRL  4
+#define WCTYPE_DIGIT  5
+#define WCTYPE_GRAPH  6
+#define WCTYPE_LOWER  7
+#define WCTYPE_PRINT  8
+#define WCTYPE_PUNCT  9
+#define WCTYPE_SPACE  10
+#define WCTYPE_UPPER  11
+#define WCTYPE_XDIGIT 12
```

Comment:
prefer enums plus optional self referential macros:

```cpp
enum {
WCTYPE_ALNUM,
#define WCTYPE_ALNUM WCTYPE_ALNUM
WCTYPE_ALPHA,
#define WCTYPE_ALPHA WCTYPE_ALPHA
...
};
```

Reply:
What you think now?

Reply:
sorry for the late response, I forgot to check this.

yes, that's fine, I wasn't aware we had this enum already, and also was under the impression it's standards-defined, but since it isn't, this is far better

thanks! :)